sh2lib/idf_component.yml

```diff
@@ -0,0 +1,6 @@
+version: "0.1.0"
+description: HTTP2 TLS Abstraction Layer
+url: https://github.com/espressif/idf-extra-components/tree/master/sh2lib
+dependencies:
+  idf: ">=5.0"
+  espressif/nghttp: ">=1.41.0"
```

Comment:
I think if this component has a dependency on nghttp >= 1.41.0, then the trick with nghttp 0.0.0 won't work, will it?

It might indeed be simpler to publish http_parser component in the component manager just for 4.x versions to use...

Reply:
We have a planned feature for the component manager to support IDF-linked dependencies and target linked, i.e. it will be possible to define a dependency like:

```
expressif/nghttp:
  version: ">=1.41.0"
  rules:
    idf: ">=5.0"
```
So it will only try to install this dependency only if the rule is satisfied. 

This feature is planned for this quarter, but I cannot say more precisely.